api/v1/swagger.yaml

```diff
@@ -13497,6 +13507,11 @@ definitions:
       botId:
         type: string
         description: UUID of the specified bot in that room
+      createdBy:
+        type: string
+        readOnly: true
+        description: >
+          Id of the User who created this roomType.
```

Comment:
Thanks @jgraff2, I will need to look into this further. When I add installedBy to BotResponse I get a response validation error when performing GET /bots because old bots do not have this field and it is null .. Going to play around with it a bit because we are not getting this error with old rooms, even though their createdBy fields are also null

Reply:
@jgraff2 not sure if you guys have any other fields in the swagger spec where the value could be null? I think we have 2 options here, unless you have any other suggestions.

1. In the swagger spec, for these installedBy and createdBy fields we can use:
`type: [ string, 'null' ]`

2. We add a migration to this PR which adds the createdBy and installedBy for existing roomTypes, rooms, botDatas & bots user to the admin user's userId